pkg/agent/flannel/setup.go

```diff
@@ -160,15 +163,15 @@ func createFlannelConf(nodeConfig *config.Node) error {
 			return err
 		}
 	case config.FlannelBackendWireguard:
-		backendConf = strings.ReplaceAll(wireguardBackend, "%flannelConfDir%", filepath.Dir(nodeConfig.FlannelConf))
+		backendConf = strings.ReplaceAll(wireguardBackend, "%flannelConfDir%", filepath.Dir(nodeConfig.FlannelConfFile))
 	default:
 		return fmt.Errorf("Cannot configure unknown flannel backend '%s'", nodeConfig.FlannelBackend)
 	}
 	confJSON = strings.ReplaceAll(confJSON, "%backend%", backendConf)
 
 	netMode, err := findNetMode(nodeConfig.AgentConfig.ClusterCIDRs)
 	if err != nil {
-		logrus.Fatalf("Error checking netMode")
+		logrus.Fatalf("Flannel error checking netMode")
 		return err
```

Comment:
nit: Fatalf causes the process to exit; we should log the error otherwise it'll be lost
```suggestion
		logrus.Fatalf("Flannel error checking netMode: %v", err)
		return err
```